christo834/CISE-assignment-1B
# Feat/submit new article

Pull-Request
Please merge feat/submit-new-article -> development branch

Test instruction:
1. In local open GitHub feat/submit-new-article branch
2. Open VS Code
3. Run backend folder
4. Run frontend folder
5. Open local host
6. Do submit an article
7. Open MongoDB see if the article is in the Collection
